posthog/models/feature_flag/feature_flag.py

```diff
@@ -174,12 +174,11 @@ def transform_cohort_filters_for_easy_evaluation(
                             # We cannot expand this cohort condition if it's not the only property in its group.
                             return self.conditions
                         try:
-                            parsed_cohort_id = str(cohort_id)
```

Comment:
you'll still need the parsing everywhere, because `prop.value` can be a string, which means we need to convert it to `int` here.

Reply:
ah, of course!